oak_functions_launcher/tests/integration_test.rs

```diff
@@ -16,29 +16,63 @@
 //! Integration tests for the Oak Functions Launcher.
 
 use lazy_static::lazy_static;
-
 use oak_functions_launcher::schema::{self, InvokeRequest};
 use oak_launcher_utils::launcher;
-use std::path::PathBuf;
+use std::{path::PathBuf, time::Duration};
 
 lazy_static! {
-    static ref WASM_PATH: PathBuf = {
-        // WORKSPACE_ROOT is set in .cargo/config.toml.
-         [env!("WORKSPACE_ROOT"),"oak_functions_launcher", "key_value_lookup.wasm"].iter().collect()
-    };
-
-    static ref LOOKUP_DATA_PATH: PathBuf = {
-        // WORKSPACE_ROOT is set in .cargo/config.toml.
-         [env!("WORKSPACE_ROOT"),"oak_functions_launcher", "mock_lookup_data"].iter().collect()
-    };
-
     static ref ENCLAVE_BINARY_PATH: PathBuf = {
         let oak_functions_linux_fd_bin_path =
-    oak_functions_test_utils::build_rust_crate_linux("oak_functions_linux_fd_bin")
-        .expect("Failed to build oak_functions_linux_fd_bin");
+            oak_functions_test_utils::build_rust_crate_linux("oak_functions_linux_fd_bin")
+                .expect("Failed to build oak_functions_linux_fd_bin");
         PathBuf::from(oak_functions_linux_fd_bin_path)
     };
+}
+
+/// Whether to skip the test. For instance, GitHub Actions does not support KVM, so we cannot run
+/// tests that require nested virtualization.
+fn skip_test() -> bool {
+    std::env::var("OAK_KVM_TESTS")
+        .unwrap_or_default()
+        .to_lowercase()
+        == "skip"
+}
 
+#[tokio::test(flavor = "multi_thread", worker_threads = 3)]
```

Comment:
Can you add a comment explaining why 3 threads are needed? 
Is there one for `oak_functions_enclave_app`, one for the launcher, and one for the client?

Reply:
It's actually for the background tasks that collect and show logs, added a comment.